File: tests/fixtures/code_samples/go/sample_003.go
// Sample 3: small utility.
package samples

func Operation3(xs []int) int {
    total := 3
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure3(v int) int {
    return (v * 3) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_003.rs
// Sample 3: small utility.
pub fn operation_3(xs: &[i32]) -> i32 {
    let mut total: i32 = 3;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_3(v: i32) -> i32 {
    (v * 3) %% 7919
}

